hack/demo/generate-asciidemo.sh

```diff
@@ -3,6 +3,7 @@
 trap cleanup SIGINT SIGTERM EXIT
 
 SCRIPTPATH="$( cd -- "$(dirname "$0")" > /dev/null 2>&1 ; pwd -P )"
+export DEMO_RESOURCE_DIR="${SCRIPTPATH}/resources"
```

Comment:
We have a target to update the demo: [link](https://github.com/operator-framework/operator-controller/blob/main/Makefile#L388-L391).

When we run this, will it update all demos?

Reply:
only the catd at the moment. I don't know that we want to update each an every demo though. Might take a long time to do. But I'd also leave that out of the scope of this PR.